scripts/get_root_dir.mjs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import path from "path";
+import url from "url";
+
+// WARNING: if you move this file, you MUST update this file path
+const RELATIVE_PATH_FROM_THIS_FILE_TO_PROJECT_ROOT = "../..";
+
+export function getRootDir(fromLocation = process.argv[1], pathToRoot = RELATIVE_PATH_FROM_THIS_FILE_TO_PROJECT_ROOT) {
+  return path.resolve(fromLocation, pathToRoot);
+}
```

Comment:
I think we are overthinking this:
```suggestion
export function getRootDir() {
  return path.resolve(__dirname, RELATIVE_PATH_FROM_THIS_FILE_TO_PROJECT_ROOT);
}
```

The extra parameters are unnecessary.

Reply:
__dirname is not valid esm, but i can generate it i think

Reply:
hopefully this works on windows we'll see